### PR TITLE
feat: use real images, add scheduled tasks

### PR DESCRIPTION
**Pull request by CalvinRodo**
Adds cloudwatch events to trigger ETL jobs
Move from hello world to actual images
No guarantee that this will work, and is still very much a WIP and testing will be required in the console.

### CODE REVIEW

File: aws/modules/ecs_task/cloudwatch.tf [hunk 1]
```diff
@@ -0,0 +1,28 @@
+resource "aws_cloudwatch_event_rule" "event_rule" {
+  name                = "${var.name}_event_rul"
```
Comment (patheard) [inline aws/modules/ecs_task/cloudwatch.tf:2]: Minor nit - spelling on `rule`

File: env/staging/ecs/task-definitions/metrics.json [hunk 2]
```diff
@@ -21,6 +21,10 @@
         }
       },
       "environment": [
+        {
+          "name": "MASKED_DATA",
```
Comment (patheard) [inline env/staging/ecs/task-definitions/metrics.json:25]: I believe this should be `MASK_DATA` for the env var name:
https://github.com/cds-snc/covid-alert-metrics/blob/main/app.py#L7

**Review by patheard — CHANGES_REQUESTED** [review]
A few comments for you.

File: env/staging/ecs/task-definitions/metrics.json (hunk 2 above)
Comment (CalvinRodo) [reply env/staging/ecs/task-definitions/metrics.json]: Fixed.

File: aws/modules/ecs_task/cloudwatch.tf (hunk 1 above)
Comment (CalvinRodo) [reply aws/modules/ecs_task/cloudwatch.tf]: Fixed.

**Review by patheard — APPROVED** [review]
Nice!  :shipit:

**Pull request merged**